pygmdata/pygmdata.py

```diff
@@ -274,7 +276,7 @@ def make_directory_tree(self, path, object_policy=None, **kwargs):
 
         self.log.debug("New file under parent OID: {}".format(oid))
         body = {
-            "action": "C",
+            "action": "U",
```

Comment:
This works even if it's C. I am not 100% sure why but I like `C`

Reply:
Agreed, it makes more sense to be `C` since it is created.

Reply:
it still says U